.github/workflows/redis-rs.yml

```
@@ -65,8 +65,18 @@ jobs:
               working-directory: ./glide-core/redis-rs/redis
 
             - name: Test
+              run: 
+              # TODO remove the concurrency limit after we fix test flakyness.
               run: |
-                  cargo test --release -- --test-threads=1 | tee ../test-results.xml
+                  versions_to_compare=$(printf "%s\n" "${{ matrix.engine.version }}" "7.1" | sort -V)
+                # Sort versions and get the latest version between the current version and 7.1, if the latest version is 7.1, skip the test.
```

```suggestion
                 # Sort versions and get the latest version between the current version and 7.1, if the latest version is 7.1, skip the test.
```
Align it, it won’t run